scripts/ci/run_perftest.sh

```diff
@@ -119,125 +139,153 @@ force_compaction() {
+      echo "unknown use-case: $1"
+      exit 1
+      ;;
+  esac
+}
 
-# Generate flux queries for field-keys and tag-values
+# Generate queries to test.
 query_files=""
-for type in field-keys tag-values; do
-  for format in http flux-http; do
-    query_fname="query-metaquery-$scale_seed_string-$type-$format.txt"
-    $GOPATH/bin/bulk_query_gen -use-case metaquery -query-type $type -format influx-$format -timestamp-start $start -timestamp-end $end -queries $queries > ${DATASET_DIR}/$query_fname
-    query_files="$query_files $query_fname"
+for format in http flux-http; do
+  # Aggregate queries
+  for usecase in window-agg group-agg bare-agg metaquery; do
+    for type in $(query_types $usecase); do
+      query_fname="${format}_${usecase}_${type}"
+      $GOPATH/bin/bulk_query_gen \
+          -use-case=$usecase \
+          -query-type=$type \
+          -format=influx-$format \
+          -timestamp-start=$(start_time $usecase) \
+          -timestamp-end=$(end_time $usecase) \
+          -queries=$queries \
+          -scale-var=$scale_var > \
+        ${DATASET_DIR}/$query_fname
+      query_files="$query_files $query_fname"
+    done
   done
 done
 
-# Load the data
-load_opts="-urls=http://${NGINX_HOST}:8086 -do-abort-on-exist=false -do-db-create=true"
-if [ -z $INFLUXDB2 ] || [ $INFLUXDB2 = true ]; then
-  load_opts="$load_opts -organization=$TEST_ORG -token=$TEST_TOKEN"
-fi
-cat ${DATASET_DIR}/$data_fname | $GOPATH/bin/bulk_load_influx $load_opts > /dev/null
+org_flag() {
+  case $1 in
+    flux-http)
+      echo -organization=$TEST_ORG
+      ;;
+    http)
+      echo -use-compatibility=true
+      ;;
+    *)
+      echo echo "unknown query format: $1"
+      exit 1
+      ;;
+  esac
+}
 
-# compacting is skipped for now on metaquery tests, which have many shards
-# force_compaction
+# Generate a DBRP mapping for use by InfluxQL queries.
+curl -XPOST -H "Authorization: Token ${TEST_TOKEN}" \
+  -d "{\"org\":\"${TEST_ORG}\",\"bucketID\":\"$(bucket_id)\",\"database\":\"$db_name\",\"retention_policy\":\"autogen\"}" \
+  http://${NGINX_HOST}:8086/api/v2/dbrps
 
 # Run the query benchmarks
 for query_file in $query_files; do
-  format=$(echo $query_file | cut -d '-' -f9)
-  format=${format%.txt}
-  query_type=$(echo $query_file | cut -d '-' -f7,8)
-  cat ${DATASET_DIR}/$query_file | ${GOPATH}/bin/query_benchmarker_influxdb --urls=http://${NGINX_HOST}:8086 --benchmark-duration=$duration --debug=0 --print-interval=0 --json=true --organization=$TEST_ORG --token=$TEST_TOKEN | jq ". += {use_case: \"metaquery\", query_type: \"${query_type}\", branch: \"${INFLUXDB_VERSION}\", commit: \"${TEST_COMMIT}\", time: \"$datestring\", i_type: \"${DATA_I_TYPE}\", query_format: \"$format\"}" > $working_dir/test-query-metaquery-$query_type-$scale_string-format-$format.json
+  format=$(echo $query_file | cut -d '_' -f1)
+  usecase=$(echo $query_file | cut -d '_' -f2)
+  type=$(echo $query_file | cut -d '_' -f3)
+  ${GOPATH}/bin/query_benchmarker_influxdb \
```

Comment:
Looks like the output of this needs some additional parsing... we currently end up with a `_field` for the title of every random query. (I think) we want to extract out just the "all queries" key from the result use its contents as the fields.